tests/tracking-tests.yml

```diff
@@ -0,0 +1,362 @@
+!TestConfig
+clip_dir: tests/test-clips
+recording_tests:
+- !TestRecording
+  device: Wallaby
```

Comment:
I think it is really interesting to know why this file was chosen.   Especially for people who are looking at the code later.  For example: 'Animal already in frame' or 'very small (inditinguishable) blob' or 'blurry insect'.   

Thus if the test fails it makes it far more motivating to fix it!

Mabye So have a summary: tag.